src/mlpack/methods/softmax_regression/softmax_regression_main.cpp

```diff
@@ -21,9 +21,9 @@ PROGRAM_INFO("Softmax Regression", "This program performs softmax regression, "
     "(-r), and if an intercept term is not desired in the model, the "
     "--no_intercept (-N) can be specified."
     "\n\n"
-    "The trained model can be saved to a file with the --output_model (-m) "
+    "The trained model can be saved to a file with the --output_model_file (-m) "
     "option.  If training is not desired, but only testing is, a model can be "
-    "loaded with the --input_model (-i) option.  At the current time, a loaded "
+    "loaded with the --input_model_file (-i) option.  At the current time, a loaded "
```

Comment:
actually for this one, it declares "input_model_file" as PARAM but uses "input_model" in the main function and documentation. So I changed everything to "input_model_file".